notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>In terms of performance, <code>df_dict = {}</code> is a little bit faster than <code>df_dict = dict()</code> since <code>{}</code> is a pre-determined literal syntax while <code>dict()</code> is a function, which requires a function call. [More info](http://katrin-affolter.ch/Python/Python_dictionaries)</p><p> </p><p>The reason why I am using <code>dict()</code> explicitly is using <code>{}</code> is sometimes confusing especially between <code>set()</code> and <code>dict()</code>.</p><p>For example, if we declare <code>a = {}</code> then it's a dictionary. However, <code>a = {0}</code> then it's a set.</p><p>That's why I've been using <code>dict()</code> instead of <code>{}</code>, for better clarification.</p><p> </p><p>Do you think it needs to be changed?</p>

Reply:
<p>This is a very good reason - I completely forgot about sets. I don't think that performance will be an issue, so your reasoning is great for keeping it as is. Thank you!</p>